style.css

```diff
@@ -0,0 +1,959 @@
+body {
+  margin: 0%;
+  padding: 0%;
+  font-family: 'Lato', sans-serif;
+}
+
+#desktop {
+  display: none;
+}
+
+#topnav {
+  display: none;
+}
+
+.mlogo1 {
+  display: none;
+}
+
+#menuOpen {
+  padding: 1rem;
+  cursor: pointer;
+}
+
+.fa-bars {
+  font-size: 2rem;
+}
+
+.fa-times {
+  font-size: 2rem;
+  padding-top: 1rem;
+}
+
+.dropDown {
+  padding-left: 1rem;
+  background-image: url(./Assets/mobileShapes.jpg);
+  background-color: rgb(197, 195, 195);
+  background-blend-mode: overlay;
+  background-position-x: 15%;
+  background-position-y: 1%;
+}
```

Comment:
- [ ] Great job on building your project as specified in the project requirement! 👏 However, there are some minor issues 🐞 that we need to address and fix. To do that kindly consider the below suggestions:
   - [ ] Kindly note that it's recommended that your mobile menu covers the entire page of your website when opened, this way your users can have a better view of it and also help them easily click on any link of their choice to navigate your website. To have this effect consider adding `position: fixed`, `width:100%`, and `height:100vh`;
   
   - [ ] After implementing this change you should your background image covering the entire page of your website this is happening because your background image is added to the dropDown class and even when the mobile menu hasn't been clicked it still shows up which shouldn't be so. To fix this issue consider moving the background image to the class name `menuList` this should help fix this issue as well.

Reply:
Done